source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -257,7 +264,12 @@ namespace PlayFab
 
         // add batch to flight tracking map
         void* customData = reinterpret_cast<void*>(batchCounter); // used to track batches across asynchronous Events API
```

Comment:
I just noticed this and it's fine to punt it from this review, but I thought I'd bring it up so you could consider addressing it in this change or filing a separate bug...

you reinterpret_cast from `size_t` to `void*` and back.
it's a bit unsafe since `size_t` and `void*` aren't guaranteed to be the same size (which is required for the transform back and forth to be absolutely safe). As a matter of fact this is exactly what the `uinptr_t` type is for! recommend replacing use of `size_t` for the batchCounter with `uintptr_t` one of these days

Reply:
there's a `reinterpret_cast<size_t>(customData)` that's still lingering around which probably needs to be changed to `static_cast<size_t>(reinterpret_cast<uintptr_t>(customData))` (the extra static cast is there because the EmitEventResponse still has it's `batch` member stored as a size_t value

Reply:
sorry looks like there is actually a second one to fix in the error callback as well

Reply:
since PlayFabEmitEventReponse is already shipped, it may break existing customers if we change the signature to be size_t, so I think we have to keep the static_casts you provided.